CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 *                        @utilitywarehouse/pubsub
 /*/customer-billing/     @utilitywarehouse/org-billing-experience
 /*/finance/              @utilitywarehouse/finance-platform
+/*/cbc/                  @utilitywarehouse/org-cashback
```

Comment:
please remove as per https://github.com/utilitywarehouse/kafka-cluster-config/tree/main/prod-aws/kafka-shared-msk#contributing

Reply:
Oh sorry, missed that! Removed!